.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
Well, then I changed my mind. I'd always gitignore .rspec-local and leave .rspec in there. And I'll start setting up my .rspec-local files to format my spec runs using the ever-so-awesome [NyanCatFormatter](https://rubygems.org/gems/nyan-cat-formatter). :grinning: